src/app/compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+  frontend:
+    image: frontend/game-recommender:latest
```

Comment:
Can we add this to our Dockerhub?

Reply:
Did you add me to it as a collaborator?

Reply:
Changed line 3 to freakynoblegas/game-recommender:frontend